presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/WindowOperatorStats.java

```diff
@@ -53,13 +53,13 @@ public static WindowOperatorStats create(WindowInfo info)
                 .filter(windowInfo -> windowInfo.getTotalRowsCount() > 0)
                 .mapToLong(DriverWindowInfo::getNumberOfIndexes)
                 .average()
-                .getAsDouble();
+                .orElse(Double.NaN);
 
         double averageNumberOfRows = info.getWindowInfos().stream()
                 .filter(windowInfo -> windowInfo.getTotalRowsCount() > 0)
                 .mapToLong(DriverWindowInfo::getTotalRowsCount)
                 .average()
-                .getAsDouble();
+                .orElse(Double.NaN);
```

Comment:
In line 80 (https://github.com/prestodb/presto/pull/10200/commits/8b643f81bc4ee6504c8f768fc5f0bf5ffea01cef#diff-cf5d7dbb1d37bd1638bd50937d1152f0R80) we're using this value in subtraction. X - NaN = NaN. This is not what we want to report. Please handle that case.